packages/synapse-react-client/src/components/JSONArrayEditor/JSONArrayEditor.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useCallback, useState } from 'react'
+import Form from '@rjsf/mui'
+import validator from '@rjsf/validator-ajv8'
+import { JSONSchema7 } from 'json-schema'
+import ArrayFieldDescriptionTemplate from '../SchemaDrivenAnnotationEditor/template/ArrayFieldDescriptionTemplate'
+import ArrayFieldItemTemplate from '../SchemaDrivenAnnotationEditor/template/ArrayFieldItemTemplate'
+import ArrayFieldTemplate from '../SchemaDrivenAnnotationEditor/template/ArrayFieldTemplate'
+import ArrayFieldTitleTemplate from '../SchemaDrivenAnnotationEditor/template/ArrayFieldTitleTemplate'
+import ButtonTemplate from '../SchemaDrivenAnnotationEditor/template/ButtonTemplate'
+import { Box, Button, Collapse, TextField } from '@mui/material'
+import { parse } from 'papaparse'
+type JSONArrayEditorProps = {
+  value: string[]
+  onChange: (newValue: string[]) => void
+  onSubmit: (formData: string[]) => void
+}
+
+const arraySchema: JSONSchema7 = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'array',
+  items: {
+    type: 'string',
+  },
+}
```

Comment:
Figure it's just easiest to use RJSF here -- we get all of the controls you could ever want for an array editor for free.